packages/v4/patternfly-docs/pages/home.js

```diff
@@ -10,269 +25,308 @@ import {
+                minHeight: "200px",
+                backgroundPosition: "center",
+                backgroundSize: "cover",
+                backgroundImage: `url(${featuredPostsData.post2.imageURL})`,
+                height: "300px",
+              }}
+            ></CardHeader>
+            <Divider />
+            <CardTitle>
+              {" "}
+              <a href={featuredPostsData.post2.URL}>
+                {featuredPostsData.post2.title}
+              </a>
+            </CardTitle>
+            <CardFooter className="pf-v5-u-color-200">
+              {featuredPostsData.post2.author} •{" "}
+              {featuredPostsData.post2.length}
+            </CardFooter>
+          </Card>
+        </GalleryItem>
+        <GalleryItem>
+          <Card>
+            <CardHeader
+              style={{
+                minHeight: "200px",
+                backgroundPosition: "center",
+                backgroundSize: "cover",
+                backgroundImage: `url(${featuredPostsData.post3.imageURL})`,
+                height: "300px",
+              }}
+            ></CardHeader>
+            <Divider />
+            <CardTitle>
+              {" "}
+              <a href={featuredPostsData.post3.URL}>
+                {featuredPostsData.post3.title}
+              </a>
+            </CardTitle>
+            <CardFooter className="pf-v5-u-color-200">
+              {featuredPostsData.post3.author} •{" "}
+              {featuredPostsData.post3.length}
+            </CardFooter>
+          </Card>
+        </GalleryItem>
+        <GalleryItem>
+          <Card>
+            <CardHeader
+              style={{
+                minHeight: "200px",
+                backgroundPosition: "center",
+                backgroundSize: "cover",
+                backgroundImage: `url(${featuredPostsData.post4.imageURL})`,
+                height: "300px",
+              }}
+            ></CardHeader>
+            <Divider />
+            <CardTitle>
+              {" "}
+              <a href={featuredPostsData.post4.URL}>
+                {featuredPostsData.post4.title}
+              </a>
+            </CardTitle>
+            <CardFooter className="pf-v5-u-color-200">
+              {featuredPostsData.post4.author} •{" "}
+              {featuredPostsData.post4.length}
+            </CardFooter>
+          </Card>
+        </GalleryItem>
+      </Gallery>
+    </PageSection>
+    <PageSection role="contentinfo">
+      <Flex flexWrap={{default: "wrap"}}>
+        <FlexItem flex={{default: "flex_1"}}>
+          <TextContent>
+            <Text component={TextVariants.h2}>Stay informed</Text>
+          </TextContent>
+        </FlexItem>
+        <Button
+          isInline
+          size="lg"
+          component="a"
+          href="https://github.com/patternfly"
+          variant="link"
+          aria-label="Patternfly on Github"
+          style={{float: "right"}}
+        >
+          Track our work on Github <ArrowRightIcon />
```

Comment:
```suggestion
          View our GitHub repositories <ArrowRightIcon />
```

Just noticed this -- since we're going to point to the top level PF github project!

Reply:
@jenny-s51 I think this remaining edit got lost in the mix!